findme-location-riddles/src/LocationRiddlesRepository.py

```diff
@@ -69,6 +69,19 @@ def get_location_riddle_by_location_riddle_id_from_db(self, location_riddle_id):
 
         return location_riddle
 
+    def update_location_riddle_rating_in_db(self, location_riddle_id, user_id, score):
+        try:
+            self.table.update_item(
+                Key={"location_riddle_id": location_riddle_id},
+                UpdateExpression="SET ratings = list_append(ratings, :i)",
+                ExpressionAttributeValues={":i": [{"user_id": user_id, "score": score}]},
```

Comment:
I would use rating instead of score to avoid confusion, also we may want to type and validate the ratings should be floats I guess from 0 to 5